--- extra-cmake-modules-5.49.0/kde-modules/KDEInstallDirs.cmake~	2018-08-04 08:36:56.000000000 +0000
+++ extra-cmake-modules-5.49.0/kde-modules/KDEInstallDirs.cmake	2018-08-28 02:23:09.784853200 +0000
@@ -641,7 +641,7 @@
 _define_absolute(SYSCONFDIR "${_default_sysconf_dir}"
     "read-only single-machine data"
     SYSCONF_INSTALL_DIR)
-_define_relative(CONFDIR SYSCONFDIR "xdg"
+_define_relative(CONFDIR DATAROOTDIR "xdg"
     "application configuration files"
     CONFIG_INSTALL_DIR)
 _define_relative(AUTOSTARTDIR CONFDIR "autostart"
